pkg/config/config.go

```diff
@@ -28,7 +28,7 @@ func ReplicationToString(r api.ReplicationConfig) string {
 }
 
 func ReplicationToConfig(dcName string, r api.ReplicationConfig) []map[string]string {
-	replicationConfig := make([]map[string]string, 1)
+	replicationConfig := make([]map[string]string, 0, 1)
```

Comment:
I cannot recall why this was `make([]map[string]string, 1)` vs `make([]map[string]string, 0)`. Out of curiosity why the change to `make([]map[string]string, 0, 1)`?

Reply:
Because []map[string]string, 1 leaves the first item as null since we use append. With 0, 1 the append will write to position 0 (using the preallocated position).